### DISCUSSION TITLE
Is there something like Cascade or Tile windows?

### DISCUSSION THREAD

**Discussion by Transisto** [rxhanson/Rectangle#720] [Q&A]
I can't seem to find it, 

I'm talking about a feature I remember from Win 3.1

![image](https://user-images.githubusercontent.com/388716/148881410-2192bdd6-65a7-4a1f-9afe-b97451da0547.png)

![image](https://user-images.githubusercontent.com/388716/148881448-50e9429b-36af-41e6-9066-4fffe8f56232.png)

**Comment by rxhanson** [comment]
Thanks for the nostalgic images!

At the moment, this is only in Rectangle Pro (https://rectangleapp.com/pro). Cascading handles all windows or all windows of an app. At this time, I only have tiling for the top 6 (2x3) or top 4 (2x2) windows.

I don't have plans to add this into Rectangle.

**Comment by adarsh** [comment]
Hello! Same question, although I tried out Rectangle Pro and couldn't get tiling to work. 

Can you please point me to the method of doing this? Thank you!

**Reply by rxhanson** [reply]
![image](https://user-images.githubusercontent.com/13651296/215820743-0ff6b7b6-2aec-4ad8-b27a-8a25713115ad.png)

Is that what you've tried?

As mentioned, there's no "tile all" option currently, although I may add that in at some point.

**Reply by adarsh** [reply]
Aha! I see it now. Thanks for the help!

**Reply by adarsh** [reply]
I've purchased Rectangle Pro but all of the default shortcuts seem different from regular Rectangle :/ I have reset them but it's still not quite right. 

My main shortcuts are:
- Cmd-Option-left (once for left half, repeat for left 2/3rds, repeat for left 1/3rd)
- (same but right)
- Cmd-Option-C (center)

And I want to add Cmd-Option-T (or something else) to tile 2/3rds left + 1/3rd right but I can't figure it out.

**Reply by rxhanson** [reply]
Thanks for purchasing. This should have properly imported from Rectangle when prompted, but there are a couple things that you can do:

Export the json config from Rectangle, import into Rectangle Pro.

or

Get the Spectacle default shortcuts in Rectangle Pro with the following terminal command:
`defaults write com.knollsoft.Hookshot alternateDefaultShortcuts -bool true`
Then restart the app and customize from there.

**Comment by adarsh** [comment]
Great, thank you!

I was also not noticing this config:
[image: image.png]


On Tue, Jan 31, 2023 at 11:33 AM Ryan Hanson ***@***.***>
wrote:

> Thanks for purchasing. This should have properly imported from Rectangle
> when prompted, but there are a couple things that you can do:
>
> Export the json config from Rectangle, import into Rectangle Pro.
>
> or
>
> Get the Spectacle default shortcuts in Rectangle Pro with the following
> terminal command:
> defaults write com.knollsoft.Hookshot alternateDefaultShortcuts -bool true
> Then restart the app and customize from there.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rxhanson/Rectangle/discussions/720#discussioncomment-4833406>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAG7CXJOFN4U6KXFYRCUQ6DWVFSG5ANCNFSM5LXYDR7A>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 

Adarsh Pandit (pronunciation guide
<https://www.youtube.com/watch?v=a4nB0QoMOhY>)
Managing Partner and Founder
Cylinder Digital
+1 313 466 2018 (mobile)

Schedule time with me using this cylinder.digital/hello
